src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -648,6 +649,15 @@ public class SimpleEvents {
 					"\tcancel the event")
 				.since("INSERT VERSION");
 		}
+		if (Skript.classExists("io.papermc.paper.event.player.PlayerInventorySlotChangeEvent")) {
+			Skript.registerEvent("Inventory Slot Change", SimpleEvent.class, PlayerInventorySlotChangeEvent.class, "[player] inventory slot chang(e|ing)")
+				.description("Called when a slot in a player's inventory is changed.", "Warning: setting the event-slot to a new item can result in an infinite loop.")
+				.requiredPlugins("Paper 1.19.2+")
+				.examples("on inventory slot change:",
+					"\tif event-item is a diamond:",
+					"\t\tsend \"You obtained a diamond!\" to player")
```

Comment:
```suggestion
				.examples(
					"on inventory slot change:",
					"\tif event-item is a diamond:",
					"\t\tsend \"You obtained a diamond!\" to player"
				)
```

Reply:
I wasn't sure as the rest of the events don't seem to do it, but should I indent the `\t` lines like how is done with `@Examples()`?

Reply:
Yes